backend/routes/subscriptionRouter.js

```diff
@@ -1,10 +1,11 @@
 const express = require("express");
-const { postSubscription, getSubscription, getPayment } = require("../controllers/subscriptionController");
+const { postSubscription, getSubscription, getPayment, subscriptionPlans } = require("../controllers/subscriptionController");
 const router = express.Router();
 
 
 router.post("/subscription", postSubscription);
 router.get("/subscription/:userName", getSubscription);
 router.get("/payment", getPayment);
+router.post("/plans",subscriptionPlans)
```

Comment:
lets make the path as `subscription-plans`

Reply:
Done.